### PR TITLE
Add sentry

### PR DESCRIPTION
**Pull request by oxkatsudon**
## General Changes

- Fixes 404 page button colors
- Adds Sentry

## Developer Notes

---

## Reviewer Checklist

Please ensure you, as the reviewer(s), have gone through this checklist to ensure that the code changes are ready to ship safely and to help mitigate any downstream issues that may occur.

- [ ]  End-to-end tests are passing without any errors
- [ ]  Code style generally follows existing patterns
- [ ]  Code changes do not significantly increase the application bundle size
- [ ]  If there are new 3rd-party packages, they do not introduce potential security threats
- [ ]  If there are new environment variables being added, they have been added to the `.env.example` file as well as the pertinant `.github/actions/*` files
- [ ]  There are no CI changes, or they have been approved by the DevOps and Engineering team(s)
- [ ]  Code changes have been quality checked in the ephemeral URL
- [ ]  QA verification has been completed
- [ ]  There are two or more approvals from the core team
- [ ]  Squash and merge has been checked

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -61,7 +59,7 @@ module.exports = withSentryConfig(
     transpileClientSDK: true,
 
     // Routes browser requests to Sentry through a Next.js rewrite to circumvent ad-blockers (increases server load)
-    tunnelRoute: "/monitoring",
+    tunnelRoute: '/monitoring',
```
Comment (0xforestwalker) [inline next.config.js:62]: What's this for @oxkatsudon ?

Comment (oxkatsudon) [reply next.config.js:62]: 🤷 
https://github.com/getsentry/sentry-wizard

Comment (0xforestwalker) [reply next.config.js:62]: What does increase server load mean? Is this applicable without a backend?

Comment (0xforestwalker) [reply next.config.js:62]: I guess it auto-falls back to not tunneling if it doesn't succeed if things are working and we have no backend? Or maybe it works on Vercel but not on prod/IPFS?

**Review by 0xforestwalker — APPROVED** [review]

**Pull request merged**